OracleWebLogic/samples/12213-domain-home-in-image-ssl/build.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+#
+#Copyright (c) 2014-2018 Oracle and/or its affiliates. All rights reserved.
```

Comment:
This copyright header is incorrect. If this script is new, it should just be "2020". If it's a modified version of an existing script, it should be the year it was originally created, a comma and then the year it was last modified, e.g. if it was created in 2018, the line would be "Copyright (c) 2018, 2020 Oracle and/or its affiliates."

Also, we no longer include the phrase "All rights reserved", so please remove it.